index.js

```diff
@@ -21,7 +21,11 @@ module.exports = function load (src, opts, cb) {
   }
 
   if (opts.text) {
-    script.appendChild(document.createTextNode(opts.text))
+    if (window.addEventListener) {
+      script.appendChild(document.createTextNode('' + opts.text))
+    } else {
+      script.text = '' + opts.text
```

Comment:
We use zuul for testing.

You run `npm run test-local` then open the browsers you want to test to the given url.

If you want to run on all the browsers, you create an https://saucelabs.com/opensauce/ account, puts your credentials like zuul says: https://github.com/defunctzombie/zuul/wiki/Zuulrc

Then `npm test`, it will run tests on many browsers. That's it.